http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -1370,10 +1370,10 @@ public ConnectionPoolDistribution(HttpClient httpClient, Processor processor)
                 Distribution distribution = new Distribution();
                 httpClient.getDestinations().stream()
                         .filter(PoolingHttpDestination.class::isInstance)
-                        .map(destination -> (PoolingHttpDestination<?>) destination)
+                        .map(destination -> (PoolingHttpDestination) destination)
```

Comment:
I wanted to make minimal changes for the upgrade so I didn't change many of these `map` calls (there are many `map` calls that I can change like that). Now you asked for that I will do it shortly.